pgmpy/models/MarkovChain.py

```diff
@@ -177,7 +177,7 @@ def add_transition_model(self, variable, transition_model):
         -----------
         variable: any hashable python object
             must be an existing variable of the model.
-        transition_model: dict
+        transition_model: dict or ndarray
```

Comment:
I think this line only tells the type of object that is expected which in this case is dict,2d-array.
so i think i could replace this with
transition_model: dict or 2d-array
The next line provides the description which i thik is apt